src/main/java/org/sagebionetworks/bridge/spring/controllers/MasterSchedulerController.java

```diff
@@ -0,0 +1,96 @@
+package org.sagebionetworks.bridge.spring.controllers;
+
+import java.util.List;
+
+import static org.sagebionetworks.bridge.Roles.ADMIN;
+import org.sagebionetworks.bridge.models.ResourceList;
+import org.sagebionetworks.bridge.models.StatusMessage;
+import org.sagebionetworks.bridge.models.TimestampHolder;
+import org.sagebionetworks.bridge.models.schedules.MasterSchedulerConfig;
+import org.sagebionetworks.bridge.services.MasterSchedulerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin
+@RestController
+public class MasterSchedulerController extends BaseController {
+    static final StatusMessage DELETED_MSG = new StatusMessage("Scheduler config deleted.");
+
+    
+    @Autowired
+    MasterSchedulerService schedulerService;
```

Comment:
By convention, this should be private, and the Autowired annotation should be on the setter.